_data/domains.yml

```diff
@@ -114,23 +113,23 @@ websites:
     img: domeneshop.png
     tfa:
       - sms
-      - software
+      - totp
     doc: https://domeneshop.no/faq?id=344
     exception: "SMS available only to customers in Norway, Sweden and Denmark."
 
   - name: Domeny.tv
     url: https://www.domeny.tv/en
     img: domenytv.png
     tfa:
-      - software
+      - proprietary
```

Comment:
![Domeny tv-2FA](https://user-images.githubusercontent.com/10876578/71980543-38192f80-3218-11ea-9f0c-4276b80da95d.png)

> Until recently, we offered you account security with the **Rublon app**. Unfortunately, due to the lack of support from the application manufacturer, we had to disable this authorization method. **Within the next few weeks** we will launch an additional authorization with **Google Authenticator**. As soon as this happens, we will immediately inform you about it via our newsletter.

\- Via [DeepL Translator](https://www.deepl.com/translator)

Should this site be marked as not having 2FA until such time they deploy TOTP 2FA?